pkg/tarmak/config/config.go

```diff
@@ -281,6 +281,30 @@ func (c *Config) AppendEnvironment(env *tarmakv1alpha1.Environment) error {
 	return c.writeYAML(c.conf)
 }
 
+func (c *Config) RemoveEnvironment(environment string) error {
+	for key, env := range c.conf.Environments {
+		if env.Name == environment {
+			c.conf.Environments = append(c.conf.Environments[:key], c.conf.Environments[key+1:]...)
+		}
+	}
+
+	done := false
```

Comment:
This could be a bit nicer, something like 
```
	for i := 0; i < len(c.conf.Clusters); i++ {
		if c.conf.Clusters[i].Environment == environment {
			c.conf.Clusters = append(c.conf.Clusters[:i], c.conf.Clusters[i+1:]...)
			i -= 1
		}
	}
```